### DISCUSSION TITLE
How to edit 'fields' in the database?

### DISCUSSION THREAD

**Discussion by xKugeki** [dani-garcia/vaultwarden#3544] [Q&A] [answered]
Hello everyone,

im actually trying to create a mysql script that edits the value of the 'fields' column in the table 'ciphers'.

But my problem is that the value inside of 'fields' is encrypted (image: https://i.imgur.com/NIKt4rH.png)

Is there any way to disable the encryption for the column 'fields' ? or can i find the secret key for the encryption?

I like it that the database is safe because of the encryption but it is hard to edit something because every value is encrypted..

Hope you can help me :)
Wish you a great day!

~ xKugeki

Extra Information:
OS: Ubuntu 22.04.2 LTS
MySQL Version: 8.0.33
Vaultwarden Version: 1.28.1

**Comment by BlackDex** [comment, marked as answer]
No there is no way to disable encryption, that would be rather useless and dangerous if that was possible.
Everything is encrypted by the clients and send encrypted to the server. The server has no way to decrypt this this.

**Reply by BlackDex** [reply]
Also, if you want to edit something, use the Bitwarden Clients, or web-vault.

**Reply by xKugeki** [reply]
Thanks for your answer :) 
Yeah, I just wanted a script that creates Tags and for that i wanted to use the fields.
I can understand why it is encrypted, because one can write stuff like password etc. in the fields - but i just wanted to find a way to let this specific column as a plain text so that i just use it for tags.

Is there any feature like tags planned for the future ? Would make searching in the vault much more easier :)

Wish you a great day!

**Reply by BlackDex** [reply]
For future plans i suggest to check the Bitwarden roadmap https://community.bitwarden.com/t/bitwarden-roadmap/12865
There even is a short video about it: https://vimeo.com/798831204

**Reply by xKugeki** [reply]
Alright, thanks again :) 
I saw it is a wished feature of the Bitwarden Community and already on their Roadmap as 'in progress'

Once again, have a nice day !
